tests/regressions/parallel/stable_merge_2964.cpp

```diff
@@ -185,7 +185,7 @@ int hpx_main(int argc, char **argv)
     HPX_TEST(equality);
 
     // Do moduler operation for avoiding overflow in ramdom_fill. (#2954)
-    int rand_base = std::rand() % 10000;
+    int rand_base = std::random_device{}() % 10000;
```

Comment:
Don't call `std:;random_device` more than once per application. Also, please use a distribution instead of modulo arithmetic for the generated numbers.

Reply:
I then have to initialize `seed` globally using `std::random_device`.